tools/RAiDER/cli/validators.py

```diff
@@ -121,20 +126,18 @@ def get_query_region(args):
             raise ValueError('Lats are out of N/S bounds; are your lat/lon coordinates switched? Should be SNWE')
         query = BoundingBox(bbox)
 
-    #TODO the next two options won't be reached currently because they are not in the aoi_group
```

Comment:
Here's a difference to what I was thinking originally and the new. For "geocoded file" I was not htinking GUNW but a geocoded DEM or LOS file. You could pass a single geocoded DEM and get all the AOI information that is needed.

Reply:
now you can add a DEM or a GUNW :D 
I think it should work for an LOS file too